### ISSUE TITLE
Error on Course 1 - Part 4 - Lesson 2 - Notebook.ipynb

### ISSUE THREAD

**Issue by davidhsv** [lmoroney/dlaicourse#113]
On exercise 2, the code is:

import tensorflow as tf
print(tf.__version__)

**mnist = tf.keras.datasets.fashion_mnist**

Therefore, importing the mnist dataset, not the fashion mnist, so all the conclusions is incorrect, we are comparing accuracy from fashion_minist 128 neurons to mnist 1024 neurons.

Thanks!

**Comment by netskip** [comment]
mnist is just a local variable.  It's still the fashion_mnist dataset.
